.github/workflows/ci.yml

```
@@ -37,11 +38,11 @@ jobs:
         chip: [esp32, esp32s2, esp32s3]
     steps:
       - uses: actions/checkout@v2
```

```suggestion
      - uses: actions/checkout@v3
```

Sorry to nitpick but let's update this action as well please, since we're bumping everything else